fabtests/pytest/efa/efa_common.py

```diff
@@ -51,3 +49,19 @@ def has_gdrcopy(hostname):
     command = "ssh {} /usr/sbin/lsmod | grep gdrdrv".format(hostname)
     process = subprocess.run(command, shell=True, check=False, stdout=subprocess.PIPE)
     return process.returncode == 0
+
+def efa_retrieve_gid(hostname):
+    """
+    determine whether a host has gdrcopy installed
```

Comment:
The docstring talks about `gdrcopy` but the function implies that you want to retrieve the gid.